src/ppx/ppx_lwt.ml

```diff
@@ -445,21 +477,21 @@ let mapper =
         lwt_log mapper fn args pexp_attributes pexp_loc
       | _ ->
         default_mapper.expr mapper expr);
-        structure_item = (fun mapper stri ->
-          default_loc := stri.pstr_loc;
-          match stri with
-          | [%stri let%lwt [%p? var] = [%e? exp]] ->
-            [%stri let [%p var] = Lwt_main.run [%e mapper.expr mapper exp]]
-
-          | {pstr_desc = Pstr_extension (({txt = "lwt"; _}, PStr [
-            {pstr_desc = Pstr_value (Recursive, _); _}]) as content, attrs); pstr_loc} ->
-            {stri with pstr_desc =
-              Pstr_extension (content, warn_let_lwt_rec pstr_loc attrs)}
-
-          | {pstr_desc = Pstr_extension (({txt = "lwt"; _}, PStr [
-            {pstr_desc = Pstr_value (Nonrecursive, vbs); _}]), _); _} ->
-            mapper.structure_item mapper (Str.value Nonrecursive (gen_top_binds vbs))
-          | x -> default_mapper.structure_item mapper x);
+    structure_item = (fun mapper stri ->
```

Comment:
If you don't mind, I'll split this whitespace change out into a separate commit before merging. You don't have to do anything (unless you really want to), I'm happy to take care of it.